### ISSUE TITLE
ckpt files

### ISSUE THREAD

**Issue by Suanmd** [EscVM/RAMS#6]
Hi! Thanks for your work!
I have another question for help. There are many different ckpt files uploaded, what models do these files represent?
I'd like to compare your methods named RAMS and RAMS+20 in the validation set you provide.
I also used the same training set and validation set. Now I need to compare all of the verification scenes.
Thanks a lot!!

**Comment by Suanmd** [comment]
Em....... I used the testing notebook to test the RAMS model but got WRONG results.
As for cPSNR in the NIR band, the Bicubic, RAMS, and RAMS++ methods should get 45.12, 48.23, and 48.51 separately, but I got 45.44, 48.17, and 48.27 in the testing notebook. As for the RED band, the problem still exists. I got 47.33/50.13/50.26 instead of 47.63/50.17/50.44.
Is there anything wrong? Thanks!

**Comment by fsalv** [comment]
Hi Suanmd,
we built and uploaded the repository several months after we finished working on the topic. Unfortunately, it happened that the actual checkpoint files we were able to recover were slightly undertrained with respect to those we used for the last submission for the challenge.

We're sorry for the inconvenience, but the results you can find in the notebook are the best reachable at present time.

**Comment by Suanmd** [comment]
Thanks a lot for your reply!&nbsp;
Your work is very excellent, and I will seriously quote it!
^-^


Suanmd
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "EscVM/RAMS"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年4月16日(星期五) 下午4:31
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [EscVM/RAMS] ckpt files (#6)





 
Hi Suanmd,
 we built and uploaded the repository several months after we finished working on the topic. Unfortunately, it happened that the actual checkpoint files we were able to recover were slightly undertrained with respect to those we used for the last submission for the challenge.
 
We're sorry for the inconvenience, but the results you can find in the notebook are the best reachable at present time.
